### PR TITLE
fix medium metrics not working

### LINKED ISSUE #83: Medium metrics are not returned
**Describe the bug**
The request to Medium is failing and no metrics are returned

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #83

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
the `GET` request to `medium` is failing with error code `403` and message: `Forbidden`, although request is working fine from browser or postman. Replacing `node-fetch` with `axios` fixed the issue so I removed `node-fetch` in app and replaced it with `axios`

File: api/networks/bounties.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const getGitcoin = async () => {
     return null
   }
 
-  const total = await response.json()
+  const total = await response.data
```
Comment (kremalicious) [inline api/networks/bounties.js:14]: just super small thing, this `await` becomes redundant with `axios` as it parses and returns JSON internally so `response.data` is simply "just there" and is not behind some promise as the `.json()` method with `fetch`

**Review by kremalicious — APPROVED** [review]
Most likely Medium wanted some HTTP headers to be sent now but their public API was always a mess as they try to keep people from using it. So whatever works here is fine.

File: api/networks/bounties.js (hunk 1 above)
Comment (KatunaNorbert) [reply api/networks/bounties.js:14]: ok, I'll remove the await

**Pull request merged**